docs/conf.py

```diff
@@ -23,7 +23,7 @@
 author = 'OpenMMLab'
 
 # The full version, including alpha/beta/rc tags
-release = '0.1.0'
+release = '0.2.1'
```

Comment:
refer to https://github.com/open-mmlab/mmcv/blob/285a052ab7fdb7f37e9b1450cb3ab63748eab601/docs/conf.py#L22